src/vellum/workflows/nodes/core/templating_node/node.py

```diff
@@ -49,7 +49,11 @@ def __new__(mcs, name: str, bases: Tuple[Type, ...], dct: Dict[str, Any]) -> Any
         if not isinstance(parent, _TemplatingNodeMeta):
             raise ValueError("TemplatingNode must be created with the TemplatingNodeMeta metaclass")
 
-        parent.__dict__["Outputs"].__annotations__["result"] = parent.get_output_type()
+        annotations = parent.__dict__["Outputs"].__annotations__
+        parent.__dict__["Outputs"].__annotations__ = {
+            **annotations,
+            "result": parent.get_output_type(),
+        }
```

Comment:
I spent some time trying to make this change higher up - in theory, every new Outputs class should be have its own `__annotations__` dict, similar to how 3.11 works. I'm deferring that for now into another ticket, as there were some other wacky errors that came up when I did so